sru-templates/manual/ec2-sru

```diff
@@ -52,6 +52,8 @@ for series in $SRU_SERIES; do
     ssh $sshopts $EC2_INST -- cloud-init status --wait --long
     ssh $sshopts $EC2_INST -- cat /run/cloud-init/result.json
     ssh $sshopts $EC2_INST -- ! grep Trace "/var/log/cloud-init.*"
+    echo "Expect IMDSv2 token headers are being used on all metadata gets. Expect 0 line count"
+    ssh $sshopts $EC2_INST --  '[[ 0 = `grep headers /var/log/cloud-init.log  | grep -v X-aws | wc -l` ]] || exit 1'
```

Comment:
grep -c <pattern> /var/log/cloud-init.log

Also, you want to use $( ) for /bin/sh vs. bash.